### ISSUE TITLE
Command to identify inconsistency between Django models and underlying databases

### ISSUE THREAD

**Issue by BoPeng** [django-extensions/django-extensions#1804]
This is a question I asked on [stack exchange](https://stackoverflow.com/questions/75427852/how-to-ensure-consistency-between-django-models-and-underlying-databases). I am posting it here since `django-extensions` looks like a perfect module to implement a solution (or already have one).

On our staging server we observed a runtime error stating a field is missing from a database,

```
column our_table.our_field does not exist
LINE 1: ...d"."type", "our_table"...
```
The field was added during a recent update with a complicated migration squashing process. It's possible that some errors were made during this process, but "manage.py showmigrations" command shows that the migration has been applied and "manage.py makemigrations" does not create any new migration. As we do not run tests on our staging or production databases, we are trying to figure out the most effective method for identifying such errors. 

In short, is there a command that can identify mismatches between the database and Django models caused by an incorrect migration like the following? 

```bash
python manage.py migrate our_app --fake
```

I suppose I am looking for something like

```bash
python manage.py check_database
```

**Comment by stefan6419846** [comment]
Shouldn't this already be handled by the `sqldiff` management command provided by `django-extensions`?

**Comment by BoPeng** [comment]
Yes, this appears to be what I am looking for. Thanks!
